pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -289,6 +289,9 @@ type MetadataServerSpec struct {
 	// The number of metadata servers that are active. The remaining servers in the cluster will be in standby mode.
 	ActiveCount int32 `json:"activeCount"`
 
+	// The number of standby metadata servers that are active.
+	ActiveStandbyCount int32 `json:"activestandbycount"`
```

Comment:
int32 seems a bit optimistic, `int16` should be enough, not to say 8...

Reply:
Okay. I will make it `int16`.

Reply:
Instead of worrying about int32 vs int16, please add CRD schema validation for this setting similar to the [activeCount](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/common.yaml#L244-L246) setting. Note that you'll need to add this validation in three places: common.yaml, resources.yaml (helm chart), and ceph_manifests.go (integration tests)